src/plugins/objects/cows/cow-plugin.ts

```diff
@@ -7,10 +7,11 @@ import { animationIds } from '@server/world/config/animation-ids';
 import { soundIds } from '@server/world/config/sound-ids';
 import { itemIds } from '@server/world/config/item-ids';
 import { objectIds } from '@server/world/config/object-ids';
+import { itemOnObjectAction } from '@server/world/actor/player/action/item-on-object-action';
 
 
-export const action: objectAction = (details) => {
-    const {player, option, objectDefinition, object} = details;
+export const action: objectAction | itemOnObjectAction = (details) => {
```

Comment:
Change this to just `: itemOnObjectAction` or `: objectAction`. A single action can not have two types, you need to define two actions if they're of a different type.